packages/web-service/src/pages/habitat/a24/common/date-validator.js

```diff
@@ -0,0 +1,54 @@
+
+import Joi from 'joi'
+import { isDate } from '../../../../common/is-date.js'
+import { isDateInvalid } from '../../../../common/is-date-invalid.js'
+
+const throwJoiError = (pageName, message, type) => {
+  throw new Joi.ValidationError('ValidationError', [{
+    message: message,
+    path: [pageName],
+    type: type,
+    context: {
+      label: pageName,
+      value: 'Error',
+      key: pageName
+    }
+  }], null)
+}
+
+export const validateDates = (payload, pageName) => {
+  const day = payload[`${pageName}-day`]
+  const month = payload[`${pageName}-month`]
+  const year = payload[`${pageName}-year`]
+
+  const badgerSeasonOpen = `05-01-${year}`
```

Comment:
If this is truly a common date validator (validating a range) then it should not mention badger setts. Best to either (a) Generalise it or (b) move it into the page directory.